src/canmatrix/canmatrix.py

```diff
@@ -338,27 +396,40 @@ def raw2phys(self, value, decodeToStr=False):
     def __str__(self):
         return self.name
 
+
 class SignalGroup(object):
     """
-    contains Signals, which belong to signal-group
+    Represents signal-group, containing multiple Signals.
     """
 
     def __init__(self, name, Id):
-        self.signals = []
+        self.signals = []  # use attr.ib?
```

Comment:
Separate PR, but sure, go for it.

Reply:
Oh ... YES. I just forgot SignalGroup while switching to attr.ib